### PR TITLE
add seata doc

### PR DESCRIPTION
**Pull request by kaori-seasons**
### Describe what this PR does / why we need it

related to [issue-2610](https://github.com/alibaba/spring-cloud-alibaba/issues/2610)

### CODE REVIEW

**Comment by steverao** [comment]
This branch cannot be rebased due to conflicts.

**Comment by kaori-seasons** [comment]
> This branch cannot be rebased due to conflicts.

solved

File: spring-cloud-alibaba-examples/seata-example/business-service/src/main/resources/application.properties [hunk 1]
```diff
@@ -0,0 +1,37 @@
+server.port=18081
```
Comment (steverao) [inline spring-cloud-alibaba-examples/seata-example/business-service/src/main/resources/application.properties]: In our origin, we use yml now.

**Review by steverao — APPROVED** [review]

File: spring-cloud-alibaba-examples/seata-example/business-service/src/main/resources/application.properties (hunk 1 above)
Comment (kaori-seasons) [reply spring-cloud-alibaba-examples/seata-example/business-service/src/main/resources/application.properties]: solved

**Comment by steverao** [comment]
Please create a same PR to branch of 2021.x.

**Pull request merged**